indexFinal.html

```diff
@@ -31,7 +31,7 @@ <h1>Schreiben Sie <br /> uns eine <br /> Nachricht</h1>
                             <legend> Anrede </legend>
                             <label> <input type="radio" name="Anrede" value="Frau"> Frau </label>
                             <label> <input type="radio" name="Anrede" value="Herr"> Herr </label>
-                            <label> <input type="radio" name="Anrede" value="Keine-Angabe"> Keine Angabe </label>
+                            <label> <input type="radio" name="Anrede" value="Keine-Angabe" id="no-gender"> Keine Angabe </label>
```

Comment:
If you want to start with a default checked radio button just add checked prop to the inpit. Noo need for javascript hacks

```suggestion
                            <label> <input type="radio" name="Anrede" value="Keine-Angabe" checked> Keine Angabe </label>
```